public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -59,12 +61,28 @@ export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, dat
                   />
                 ) : null}
                 <h2 className={styles.titleText}>{title}</h2>
-                <Icon name="angle-down" className="panel-menu-toggle" />
-                <PanelHeaderMenuWrapper panel={panel} dashboard={dashboard} show={panelMenuOpen} onClose={closeMenu} />
-                {data.request && data.request.timeInfo && (
-                  <span className="panel-time-info">
-                    <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
-                  </span>
+                {!getConfig().isPublicDashboardView ? (
+                  <div data-testid="panel-dropdown">
+                    <Icon name="angle-down" className="panel-menu-toggle" />
+                    <PanelHeaderMenuWrapper
+                      panel={panel}
+                      dashboard={dashboard}
+                      show={panelMenuOpen}
+                      onClose={closeMenu}
+                    />
+                    {data.request && data.request.timeInfo && (
+                      <span className="panel-time-info">
+                        <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
+                      </span>
+                    )}
+                  </div>
+                ) : (
+                  data.request &&
+                  data.request.timeInfo && (
+                    <span className="panel-time-info">
+                      <Icon name="clock-nine" size="sm" /> {data.request.timeInfo}
+                    </span>
+                  )
```

Comment:
Why is this inside the dropdown and outside? This ? logic is not needed, just have it outside the getConfig().isPublicDashboardView conditional.  panel-time-info is absolutely positioned so it works either way. 

You added the div ` <div data-testid="panel-dropdown">` , the pane-time-info can be outside it.